_config.yml

```diff
@@ -21,7 +21,7 @@ description: > # this means to ignore newlines until "baseurl:"
   Documentation for developers integrating FIO protocol.
 
 # Base url. Leave blank if you don't want the extra path.
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "/fio-devhub-testnewOrg" # the subpath of your site, e.g. /blog
```

Comment:
You should only need to change this if you are using your own fork. Or maybe you did not pull latest from master? Need to remove this.

Reply:
I removed... (I'd added it back in to get my fork working with the onboarding thing.)